### PR TITLE
Chores/libocr upgrade

### PR DESCRIPTION
**Pull request by sdrug**

### CODE REVIEW

**Review by aalu1418 — DISMISSED** [review]
🚀

File: pkg/terra/report.go [hunk 1]
```diff
@@ -87,14 +87,18 @@ func (c ReportCodec) BuildReport(oo []median.ParsedAttributedObservation) (types
...
-	jBytes, err := ToBytes(juelsPerFeeCoin, juelsPerFeeCoinSizeBytes)
+	jBytes, err := bigbigendian.SerializeSigned(juelsPerFeeCoinSizeBytes, juelsPerFeeCoin)
 	if err != nil {
 		return nil, err
 	}
 	report = append(report, jBytes...)
 	return report, nil
 }
 
+func (c ReportCodec) MaxReportLength(n int) int {
```
Comment (kaleofduty) [inline pkg/terra/report.go:98]: let's add a test for this please

**Review by kaleofduty — DISMISSED** [review]

File: pkg/terra/report_test.go [hunk 2]
```diff
@@ -136,6 +136,7 @@ func TestMedianFromReport(t *testing.T) {
 			}
 			report, err := cdc.BuildReport(pos)
 			require.NoError(t, err)
+			assert.Equal(t, len(report), cdc.MaxReportLength(len(tc.obs)))
```
Comment (kaleofduty) [inline pkg/terra/report_test.go:155]: Also test with n up to 31?

**Comment by kaleofduty** [comment]
looks good, but tests are failing?

**Comment by aalu1418** [comment]
> looks good, but tests are failing?

e2e tests won't pass right now because of the added method, so the core node can't build. once we update core (with up to date terra & solana packages), the tests should pass again

**Review by aalu1418 — APPROVED** [review]

**Pull request merged**